### PR TITLE
10th weeks

### PR DESCRIPTION
**Pull request by AgwaB**

### CODE REVIEW

File: express/Influanswer/config/s3multer.js [hunk 1]
```diff
@@ -5,17 +5,29 @@ aws.config.loadFromPath('./config/s3config.json');
...
         key: function(req, file, cb) {
             cb(null, Date.now() + '.' + file.originalname);
         }
     })
 });
 
-module.exports ={
-    uploadContentImage : uploadContentImage
-};
+const campaignImageUpload = multer({
```
Comment (leesd556) [inline express/Influanswer/config/s3multer.js]: test

Comment (leesd556) [inline express/Influanswer/config/s3multer.js]: test

**Review by AgwaB — COMMENTED** [review]
test

File: express/Influanswer/config/s3multer.js [hunk 2]
```diff
@@ -5,17 +5,29 @@ aws.config.loadFromPath('./config/s3config.json');
...
+    storage: multerS3({
+        s3: s3,
+        bucket: 'influanswer/campaign',
+        acl: 'public-read-write',
+        key: function(req, file, cb) {
+            cb(null, Date.now() + '.' + file.originalname);
+        }
+    })
+});
```
Comment (leesd556) [inline express/Influanswer/config/s3multer.js]: test

File: express/Influanswer/config/s3multer.js (hunk 1 above)
Comment (leesd556) [inline express/Influanswer/config/s3multer.js]:
```suggestion
const campaignImageUpload = multer({
```

File: express/Influanswer/config/s3multer.js (hunk 2 above)
Comment (AgwaB) [inline express/Influanswer/config/s3multer.js:28]: test

File: express/Influanswer/config/s3multer.js [hunk 3]
```diff
@@ -5,17 +5,29 @@ aws.config.loadFromPath('./config/s3config.json');
...
-module.exports ={
-    uploadContentImage : uploadContentImage
-};
+const campaignImageUpload = multer({
+    storage: multerS3({
+        s3: s3,
+        bucket: 'influanswer/campaign',
+        acl: 'public-read-write',
+        key: function(req, file, cb) {
+            cb(null, Date.now() + '.' + file.originalname);
```
Comment (AgwaB) [inline express/Influanswer/config/s3multer.js:25]: test

File: express/Influanswer/config/s3multer.js (hunk 1 above)
Comment (AgwaB) [inline express/Influanswer/config/s3multer.js:19]: test3

Comment (AgwaB) [reply express/Influanswer/config/s3multer.js:19]: d

**Review by AgwaB — COMMENTED** [review]
test

File: express/Influanswer/config/s3multer.js [hunk 4]
```diff
@@ -5,17 +5,29 @@ aws.config.loadFromPath('./config/s3config.json');
...
+        }
+    })
+});
+
+module.exports = {
+  registerImageUpload : registerImageUpload,
+  campaignImageUpload : campaignImageUpload
+}
```
Comment (leesd556) [inline express/Influanswer/config/s3multer.js]:
```suggestion
module.exports = {
  registerImageUpload : registerImageUpload,
  campaignImageUpload : campaignImageUpload
}
```

Comment (leesd556) [inline express/Influanswer/config/s3multer.js]:
```suggestion
}
```

File: express/Influanswer/config/s3multer.js [hunk 5]
```diff
@@ -5,17 +5,29 @@ aws.config.loadFromPath('./config/s3config.json');
...
+        s3: s3,
+        bucket: 'influanswer/campaign',
+        acl: 'public-read-write',
+        key: function(req, file, cb) {
+            cb(null, Date.now() + '.' + file.originalname);
+        }
+    })
+});
+
+module.exports = {
```
Comment (AgwaB) [inline express/Influanswer/config/s3multer.js:30]:
```suggestion
```

**Review by AgwaB — COMMENTED** [review]
test4

**Review by AgwaB — COMMENTED** [review]
```go
func() test {
}
```

**Review by AgwaB — COMMENTED** [review]
```go
func() test {
}
```

`test`

File: express/Influanswer/config/s3multer.js [hunk 6]
```diff
@@ -5,17 +5,29 @@ aws.config.loadFromPath('./config/s3config.json');
...
 });
 
-module.exports ={
-    uploadContentImage : uploadContentImage
-};
+const campaignImageUpload = multer({
+    storage: multerS3({
+        s3: s3,
+        bucket: 'influanswer/campaign',
+        acl: 'public-read-write',
```
Comment (AgwaB) [inline express/Influanswer/config/s3multer.js:23]:
```suggestion
        ~~~
        ~~~
```

**Review by AgwaB — COMMENTED** [review]
`dsadasd
dsadsad
`

**Review by AgwaB — COMMENTED** [review]
```
sdafsadf
sadf
sdafs
af
s
```

**Pull request merged**